main.py

```diff
@@ -1,7 +1,63 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(
+        name="Western",
+    )
+    Genre.objects.create(
+        name="Action",
+    )
+    Genre.objects.create(
+        name="Dramma",
+    )
+    Actor.objects.create(
+        first_name="George",
+        last_name="Klooney"
+    )
+    Actor.objects.create(
+        first_name="Kianu",
+        last_name="Reaves"
```

Comment:
Typographical Error: Correct the actor's first name from "Kianu" to "Keanu" and last name from "Reaves" to "Reeves" initially to avoid unnecessary updates later.